### DISCUSSION TITLE
Opening Ports

### DISCUSSION THREAD

**Discussion by TheeKia** [WeAreMahsaAmini/FreeInternet#75] [Q&A]
Step 3.3 on **protocols**/**shadowsocks-v2ray-tls**
`Open ports 80 TCP ingress and 443 TCP ingress on your server (security group/firewall)`
Can you explain how we can do it?
I'm trying to setup new Shadowsocks but I'm not very familiar with these concepts.

**Comment by TheeKia** [comment]
> Step 3.3 on **protocols**/**shadowsocks-v2ray-tls** `Open ports 80 TCP ingress and 443 TCP ingress on your server (security group/firewall)` Can you explain how we can do it? I'm trying to setup new Shadowsocks but I'm not very familiar with these concepts.

I ignored this line since I didn't know what to do. But it worked.
Thanks for the setup guide

**Comment by 0xland** [comment]
some server providers by default blocks all ports and you need to open them manually.
